packages/orchestrator-ui-components/src/components/WfoForms/formFields/deprecated/IpNetworkField.tsx

```diff
@@ -48,17 +45,18 @@ function IpNetwork({
 }: IPvAnyNetworkFieldProps) {
     const { formRowStyle } = useWithOrchestratorTheme(getCommonFormFieldStyles);
 
-    const t = useTranslations('pydanticForms');
-    const [selectedPrefix, setSelectedPrefix] = useState<IpBlock | undefined>(
-        undefined,
-    );
-    const [manualOverride, setManualOverride] = useState(false);
+    // The state is needed in order to keep the selected prefix in the SplitPrefix component
+    const [selectedPrefix] = useState<string | undefined>(value);
 
-    const usePrefix = selectedPrefix?.prefix ?? value;
+    const usePrefix = selectedPrefix;
     const [subnet, netmask] = usePrefix?.split('/') ?? ['', ''];
+
+    const netMaskInt = parseInt(netmask, 10);
     const usedPrefixMin =
         prefixMin ??
-        parseInt(netmask, 10) + (selectedPrefix?.state === 0 ? 0 : 1);
+        (netMaskInt < 32 && name === 'ip_sub_prefix'
+            ? netMaskInt + 1
+            : netMaskInt);
```

Comment:
maybe consider, in a separate statement, to test for NaN

Reply:
fixed